generators/app/templates/cmd/sonic/sonic.go

```diff
@@ -4,7 +4,10 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/openware/igonic/config"
+  "github.com/foolin/goview/supports/ginview"
+	"github.com/gin-gonic/gin"
+	"<%= git_domain %>/<%= organization %>/<%= project %>/config"
+	"<%= git_domain %>/<%= organization %>/<%= project %>/routes"
```

Comment:
I suggest making a function or simply a prop : <%= package_name %>
project name being the last element of the url

Reply:
done